### DISCUSSION TITLE
Unable to see cup profile on report

### DISCUSSION THREAD

**Discussion by fcebreros** [artisan-roaster-scope/artisan#1563] [Q&A] [answered]
Hi, 
I'm able to save the cup profile but it does not appear on the roast report.
On the roast report its empty. 
I'm on windows 10. I have tried it on a computer with the last version of Artisan
and on another with the 2.8.4 version and it was the same.
Anyone has had the same problem?
Greetings
FC

**Comment by MAKOMO** [comment, marked as answer]
Thanks for reporting! This is a regression and should be fixed with my latest commit. New [continuous builds](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) are available containing this fix.

**Reply by fcebreros** [reply]
Great!
Danke!
Tested and it works. 
I have noticed that on the PDF export the spider-graph is now on the next page
and not directly under the curve. (not a big deal)
Best regards
FC
